P0009/P0009.bs

```diff
@@ -1802,7 +1804,7 @@ Table �: Accessor policy requirements
   <td>*Returns:* A standard pointer to the contiguous set of objects accessed by `p`.</td>
 </tr>
 <tr>
-  <td>`A::deref(p,i)`</td>
+  <td>`A::access(p,i)`</td>
```

Comment:
This needs to be `a.access(p, i)` (and the other two also need to be phrased in the form of member function semantics).